joint_calling/upload_processor.py

```diff
@@ -69,9 +73,14 @@ def batch_move_files(
     jobs = []
     for sample in files:
         previous_location = os.path.join('gs://', source_prefix, sample)
-        new_location = os.path.join('gs://', destination_prefix, sample)
 
-        j = batch.new_job(name=f'move {sample}')
+        # Get internal sample ID
+        sapi = SampleApi()
+        internal_id = sapi.get_internal_id(sample)  # TO IMPLEMENT: API CALL
```

Comment:
It might be quicker to batch these calls at the start, eg:

```python
internal_id_map = sapi.get_internal_ids_from_external_ids(samples)
for external_sample_id in samples:
    internal_id = internal_id_map[external_sample_id]
```

Reply:
Great point! Pulled it out :)